src/NodaTime.Web/Markdown/root/versions.md

```diff
@@ -708,3 +711,16 @@ Bug fixes:
 ## 1.0.0-beta1, released 2012-04-12 with tzdb 2012c
 
 - Initial beta release
+
+# <a name="tzdb-updates"></a>Update policy for time zone data
```

Comment:
prefer `<a id="">` to `<a name="">`.

(Also, I assume this actually DTRT in the rendered version?)

Reply:
Didn't know about this. Will check both forms.